Docs/Ai_help.md

```diff
@@ -0,0 +1,18 @@
+# AI Section
+
+## AI Help
+
+In this section we will go over the AI help that has been through out the process of creating the project. This will be about what AI help has been used and to what extend.
+
+### ChatGBT
+
+ChatGBT is one of the AI's that we have used through out the project for smaller questions. Theses questions range from C# code, refactoring of code, setup of docker or .NET projects. With the help from the AI we can ask questions and get quick response that can help us in the right direction. There are somethings to consider when using ChatGBT the main thing will be the validity of the response since we can't guarantee that it's right.
+
+### Github CoPilot
+
+Github CoPilot has been used through out the project to help with speeding up the process of writing code. It gives suggestions to code that you were writing either way with autocompleting a few lines. It can also sometimes help with debugging. It has also help with the creation of unit tests.
```

Comment:
I think it will be a good idea, to elaborate more on the point that we have used it for repetitive task like writing a lot of unit tests and what we have used it for, instead of focusing on what it can do,